physionet-django/static/custom/css/form-control-input.css

```diff
@@ -41,3 +41,7 @@ input::placeholder {
   color: #868e96;
   opacity: 1;
 }
+
+.btn-search{
```

Comment:
Is this empty style here intentionally? If so, please could you explain its purpose?